ios/CodePush/CodePush.h

```diff
@@ -109,8 +111,18 @@ failCallback:(void (^)(NSError *err))failCallback;
 + (void)copyEntriesInFolder:(NSString *)sourceFolder
                  destFolder:(NSString *)destFolder
                       error:(NSError **)error;
+
 + (NSString *)findMainBundleInFolder:(NSString *)folderPath
                                error:(NSError **)error;
+
++ (NSString *)getDefaultAssetsFolderName;
++ (NSString *)getDefaultJsBundleName;
```

Comment:
Are either of these two methods currently used outside of its enclosing class? If not, I'd rather omit them from the interface.